records_mover/records/delimited/conversions.py

```diff
@@ -39,16 +39,24 @@
 # account for MM/DD time.
 #
 # https://github.com/bluelabsio/records-mover/issues/75
-#
+
+# TODO: Need to make these functions or this will be very brittle.
+# Add test cases to repro, verify it gets reproed, and then fix.
 python_date_format_from_hints: Dict[Union[HintDateFormat, Literal['DD/MM/YY']], str] = {
+    'DD-MM-YYYY': '%d-%m-%Y',
+    'MM-DD-YYYY': '%m-%d-%Y',
     'YYYY-MM-DD': '%Y-%m-%d',
-    'MM/DD/YY': '%m/%d/%Y',
-    'DD/MM/YY': '%d/%m/%Y',
+    'MM/DD/YY': '%m/%d/%y',
+    'DD/MM/YY': '%d/%m/%y',
+    'DD-MM-YY': '%d-%m-%y',
 }
 
+# TODO: Need to make these functions or this will be very brittle.
+# Add test cases to repro, verify it gets reproed, and then fix.
 python_time_format_from_hints: Dict[HintTimeOnlyFormat, str] = {
     'HH24:MI:SS': '%H:%M:%S',
-    'HH12:MI AM': '%I:%M:%S %p',
+    'HH:MI:SS': '%H:%M:%S',
+    'HH12:MI AM': '%I:%M %p',
 }
```

Comment:
This is another example of something which will need to be converted from mappings to full functions.